project01/paulo/game_canvas.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <cinder/gl/gl.h>
+
+#include <vector>
+
+class GameCanvas
+{
+public:
+  GameCanvas(ci::ivec2 /*canvasDelta*/, int /*width*/, int /*height*/, int /*pointSize*/);
+
+  void Clear();
+  void Add(const ci::ivec2&, const ci::Color&);
+  void ShowMessage(const std::string& /*message*/, const ci::Color&) const;
+
+  int GetWidth() const { return mWidth; }
+  int GetHeight() const { return mHeight; }
+
+  ci::ivec2 GetRandomUnusedPoint() const;
```

Comment:
Not sure if it is nice to give this responsibility to a Canvas. Seems like a smart canvas ;)